### ISSUE TITLE
[ListView] swallows render error

### ISSUE THREAD

**Issue by fabiomcosta** [facebook/react-native#1965]
When an error happens inside `ListView`'s `renderRow` property the error is swallowed and never shown on the RedBox.
Ex:
```jsx
<ListView
  renderRow={() => <TouchableHighlight><Text>a</Text><Text>b</Text></TouchableHighlight>}
  dataSource={dataSource}
/>
```
Because `TouchableHighlight` accepts only one child, there should be an error thrown, but what happens is that no item from the dataSource is ever rendered.

I'm using v0.7.1

**Comment by brentvatne** [comment]
@fabiomcosta - do you have a chance to submit a PR for this? It would be great :smile:

**Comment by boourns** [comment]
While debugging a similar error, I tried reproducing this error using the example given in the PR description.  It looks like it's now fixed on react-native 0.11.0 and errors are not swallowed:

`index.ios.js`:
```javascript
var ListViewBug = React.createClass({
  render: function() {
    var ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});

    return (
      <ListView
        renderRow={() => <TouchableHighlight><Text>a</Text><Text>b</Text></TouchableHighlight>}
        dataSource={ds.cloneWithRows(['row 1', 'row 2'])}
      />
    );
  }
});
```

output (as expected):
![image](https://cloud.githubusercontent.com/assets/699550/10004202/7b2bea12-607e-11e5-897d-9c7bc61a22cf.png)

so @brentvatne I think this issue can be closed now
